core/domain/state_domain.py

```diff
@@ -946,6 +986,30 @@ def __init__(self, value, schema):
         self.value = value
         self.schema = schema
 
+    def _register_all_translatable_fields(self) -> None:
+        """Registers all of the translatable fields/objects in the interaction
+        customization args.
+        """
+
```

Comment:
Remove extra newline here (ditto in all other places)

Reply:
Removed.